app/src/components/notifier/Notifier.test.js

```diff
@@ -0,0 +1,178 @@
+  input.simulate('keypress', {key: 'Enter'});
+
+  //check for expected state of the store
+  const nowStore = store.getState();
+  expect(nowStore.notify.msg).toEqual('');
+
+  //Check for class name of Notifier Snack Bar. Present now.
+  expect(wrapper.find('.Notifier-root-120').hostNodes().length).toEqual(0);
+});
+
+it('Notifier can open snackbar for empty query', () => {
+
+  const SEARCH_STR = '';
+
+  const wrapper = mount(
+    <Provider store={store}>
+      <MemoryRouter>
+        <App/>
+      </MemoryRouter>
+    </Provider>);
+
+  //Check for class name of Notifier Snack Bar. Initially not present.
+  expect(wrapper.find('.Notifier-root-120').hostNodes().length).toEqual(0);
+
+  let input = wrapper.find('input').last();
+  input.simulate('change', { target: { value: SEARCH_STR}});
+  //and try to submit the query
+  input.simulate('keypress', {key: 'Enter'});
+
+  //check for expected state of the store
+  const nowStore = store.getState();
+  expect(nowStore.notify.msg).toEqual(QUERY_LEN_ERR);
+
+  //Check for class name of Notifier Snack Bar. Present now.
+  expect(wrapper.find('.Notifier-root-120').hostNodes().length).toEqual(1);
+});
+
+it('Notifier can open snackbar for short query', () => {
+
+    const SEARCH_STR = 'f';
+
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter>
+          <App/>
+        </MemoryRouter>
+      </Provider>);
+
+    //Check for class name of Notifier Snack Bar. Initially not present.
+    expect(wrapper.find('.Notifier-root-120').hostNodes().length).toEqual(0);
+
+    let input = wrapper.find('input').last();
+    input.simulate('change', { target: { value: SEARCH_STR}});
+    //and try to submit the query
+    input.simulate('keypress', {key: 'Enter'});
+
+    //check for expected state of the store
+    const nowStore = store.getState();
+    expect(nowStore.notify.msg).toEqual(QUERY_LEN_ERR);
+
+    //Check for class name of Notifier Snack Bar. Present now.
+    expect(wrapper.find('.Notifier-root-120').hostNodes().length).toEqual(1);
+});
+
+it('Notifier can open snackbar for errors returned from httpService', () => {
+
+  const wrapper = mount(
+    <Provider store={store}>
+      <MemoryRouter>
+        <App/>
+      </MemoryRouter>
+    </Provider>);
+
+  //Check for class name of Notifier Snack Bar. Initially not present.
+  expect(wrapper.find('.Notifier-root-120').hostNodes().length).toEqual(0);
+
+  let dummyUrl = "http://katlas.com/v1/qsl";
+  let dummyParams = {qslstring: 'Cluster'};
+  const response = {
+    body: {},
+    status: 400
+  };
+
+  fetchMock.get('*', response, { overwriteRoutes: true });
+
+  return HttpService.get({
+    url: dummyUrl,
+    params: dummyParams
+  }).then((response) => {
+    expect(fetchMock.called).toBeTruthy();
+    expect(response).toEqual(null);
+
+    //TODO - SUSH - Action invoked but store is getting empty value here.
+    //check for expected state of the store
+    //const nowStore = store.getState();
```

Comment:
actually, this is the section which you mentioned you're still trying to get working before merging the PR, correct? if so, ignore my request to remove the lines... since you'll eventually be un-commenting them to get it working anyways

Reply:
@davemasselink  yes, will sync up with you when avl on this one.

Reply:
ok, great... I'm hoping I'll be feeling well enough to be in the office Friday. we can always try for a bluejeans chat and screen share, if helpful, as well